packages/jest-resolve/package.json

```diff
@@ -9,9 +9,10 @@
   "main": "build/index.js",
   "dependencies": {
     "browser-resolve": "^1.11.2",
-    "chalk": "^2.0.1"
+    "chalk": "^2.0.1",
+    "jest-util": "^21.2.1"
```

Comment:
Unfortunately `jest-resolve` must not rely on `jest-util`. We are going to make use of `jest-resolve` in https://github.com/facebook/metro soon and this would pull in too much unnecessary stuff. Can we just inline the realpath function instead?

Reply:
should probably create a module with the realpath function by itself. I've copied it in a personal project already

Reply:
#5097